moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -568,15 +568,17 @@ void JogCalcs::enforceJointVelocityLimits(Eigen::VectorXd& calculated_joint_chan
 bool JogCalcs::checkIfJointsWithinURDFBounds(trajectory_msgs::JointTrajectory& new_jt_traj)
 {
   bool halting = false;
+
   for (auto joint : joint_model_group_->getJointModels())
   {
     if (!kinematic_state_->satisfiesVelocityBounds(joint))
     {
       ROS_WARN_STREAM_THROTTLE_NAMED(2, LOGNAME, ros::this_node::getName() << " " << joint->getName() << " "
                                                                            << " close to a "
                                                                               " velocity limit. Enforcing limit.");
+
       kinematic_state_->enforceVelocityBounds(joint);
-      for (std::size_t c = 0; c < num_joints_; ++c)
+      for (std::size_t c = 0; c < new_jt_traj.points[0].velocities.size(); ++c)
```

Comment:
Small simplification / homogenization:
```suggestion
      for (std::size_t c = 0; c < new_jt_traj.joint_names.size(); ++c)
```